src/rego/lib_exclude_update/lib_exclude_update.rego

```diff
@@ -0,0 +1,7 @@
+package lib.exclude_update
+
+import future.keywords.in
```

Comment:
nit: If we aren't using `in` we should probably drop from all `lib.exclude_update`?

Reply:
Good catch! I removed all `future.keywords.in` import.